src/pcapi/templates/admin/confirm_suspension.html

```diff
@@ -0,0 +1,30 @@
+{% extends 'admin/master.html' %}
+{% block body %}
+<div class="alert alert-danger" role="alert">
+  <strong>Attention !</strong> Vous êtes sur le point de <strong>suspendre</strong> le compte utilisateur suivant :
+</div>
+
+<p>
```

Comment:
```suggestion
<div>
```
Je pense que syntaxiquement ce sera quand même un peu mieux :)